File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/hyperbolic/SinhZ.java
package net.fwitz.math.main.binary.complex.functions.hyperbolic;

import net.fwitz.math.binary.complex.Complex;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class SinhZ {
    public static void main(String[] args) {
        new ComplexFunctionPlot("sinh(z)")
                .computeFn(Complex::sinh)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/hyperbolic/SinhZ.kt
package net.fwitz.math.main.binary.complex.functions.hyperbolic

import net.fwitz.math.binary.complex.Complex
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object SinhZ {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("sinh(z)")
        .computeFn(Complex::sinh)
        .render()
}